Client/Frontend/Autofill/CreditCard/CreditCardBottomSheet/SingleCreditCardView.swift

```diff
@@ -0,0 +1,291 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Shared
+import Storage
+import Common
+
+import Foundation
+import UIKit
+
+class SingleCreditCardViewController: UIViewController, UITableViewDelegate, UITableViewDataSource, BottomSheetChild, Themeable {
+    // MARK: UX
+    struct UX {
+        static let containerPadding: CGFloat = 18.0
+        static let tableMargin: CGFloat = 0
+        static let distanceBetweenHeaderAndTop: CGFloat = UIDevice.current.userInterfaceIdiom == .pad ? 0 : -10
+        static let distanceBetweenButtonAndTable: CGFloat = UIDevice.current.userInterfaceIdiom == .pad ? 18 : 34
+        static let distanceBetweenHeaderAndCells: CGFloat = 24
+        static let yesButtonCornerRadius: CGFloat = 13
+        static let yesButtonFontSize: CGFloat = 16.0
+        static let yesButtonHeight: CGFloat = 45.0
+        static let bottomSpacing: CGFloat = 32.0
+        static let buttonsSpacing: CGFloat = 8.0
+        static let contentViewWidth: CGFloat = UIDevice.current.userInterfaceIdiom == .pad ? 545 : 339
+        static let headerPreferredHeight: CGFloat = UIDevice.current.userInterfaceIdiom == .pad ? 64 + 24 : 84 + 24
```

Comment:
Why the math there? Ex: `64 + 24 : 84 + 24`. Maybe a small comment explaining what this does?